diff --git a/lua/chadrc.lua b/lua/chadrc.lua
index 1d841e9..5c72540 100644
--- a/lua/chadrc.lua
+++ b/lua/chadrc.lua
@@ -5,7 +5,7 @@
 local M = {}
 
 M.ui = {
-	theme = "onedark",
+	theme = "rosepine-dawn",
 
 	-- hl_override = {
 	-- 	Comment = { italic = true },
diff --git a/lua/configs/lspconfig.lua b/lua/configs/lspconfig.lua
index a73657a..e7ace5f 100644
--- a/lua/configs/lspconfig.lua
+++ b/lua/configs/lspconfig.lua
@@ -4,7 +4,7 @@ local on_init = require("nvchad.configs.lspconfig").on_init
 local capabilities = require("nvchad.configs.lspconfig").capabilities
 
 local lspconfig = require "lspconfig"
-local servers = { "html", "cssls" }
+local servers = { "html", "cssls", "clangd", "pyright", "cmake" }
 
 -- lsps with default config
 for _, lsp in ipairs(servers) do
diff --git a/lua/plugins/init.lua b/lua/plugins/init.lua
index e62d896..4f11e44 100644
--- a/lua/plugins/init.lua
+++ b/lua/plugins/init.lua
@@ -8,31 +8,39 @@ return {
   },
 
   -- These are some examples, uncomment them if you want to see them work!
-  -- {
-  --   "neovim/nvim-lspconfig",
-  --   config = function()
-  --     require("nvchad.configs.lspconfig").defaults()
-  --     require "configs.lspconfig"
-  --   end,
-  -- },
-  --
-  -- {
-  -- 	"williamboman/mason.nvim",
-  -- 	opts = {
-  -- 		ensure_installed = {
-  -- 			"lua-language-server", "stylua",
-  -- 			"html-lsp", "css-lsp" , "prettier"
-  -- 		},
-  -- 	},
-  -- },
-  --
-  -- {
-  -- 	"nvim-treesitter/nvim-treesitter",
-  -- 	opts = {
-  -- 		ensure_installed = {
-  -- 			"vim", "lua", "vimdoc",
-  --      "html", "css"
-  -- 		},
-  -- 	},
-  -- },
+  {
+    "neovim/nvim-lspconfig",
+    config = function()
+      require("nvchad.configs.lspconfig").defaults()
+      require "configs.lspconfig"
+    end,
+  },
+
+  {
+  	"williamboman/mason.nvim",
+  	opts = {
+  		ensure_installed = {
+  			"lua-language-server", "stylua",
+  			"html-lsp", "css-lsp" , "prettier",
+        "clangd", "clang-format",
+        "cmake-language-server", "cmakelang",
+        "pyright", "flake8", "black",
+  		},
+  	},
+  },
+
+  {
+  	"nvim-treesitter/nvim-treesitter",
+  	opts = {
+  		ensure_installed = {
+  			"vim", "lua", "vimdoc",
+        "html", "css",
+        "python",
+        "c", "cpp", "cmake",
+        "go",
+        "dockerfile",
+        "csv"
+  		},
+  	},
+  },
 }
